cache/disk/lru.go

```diff
@@ -27,16 +27,20 @@ type SizedLRU interface {
 	Get(key Key) (value sizedItem, ok bool)
 	Remove(key Key)
 	Len() int
-	CurrentSize() int64
+	CurrentSize() int64 // Returns the current used + reserved size.
```

Comment:
Rename this method to CommittedAndReservedSize()? Might be interesting to export both as stats?

Reply:
I wonder what is most useful form? This is only used by `Stats()` and tests. A pair of `<current used + reserved size>, <reserved size>` seems reasonable and won't confuse existing users who are collecting and graphing "CurrSize".

Callers could use `totalSize, reservedSize := c.CurrentSize()` which is pretty readable IMO.

Reply:
Renamed to `TotalSize()` and added `ReservedSize()`. I left the label in the json output as `CurrSize`.